clients/mobile/src/donationWorkflow/createUpdateDonation/Donation.tsx

```diff
@@ -91,7 +90,7 @@ const CreateBloodRequest = () => {
             />
           </View>
 
-          <View style={styles.fieldSpacing}>
+          <View style={[styles.fieldSpacing]}>
             <Dropdown
               label='Select City'
               isRequired={true}
```

Comment:
wouldn't isRequired be enough

Reply:
@sakibstark11 Bhai I don't have enough context behind this variable usage...